package.json

```diff
@@ -61,7 +61,7 @@
     "jasmine-core": "^3.3.0",
     "jasmine-jquery": "^2.1.1",
     "jasmine-spec-reporter": "^4.2.1",
-    "looks-same": "^5.0.1",
+    "looks-same": "^6.0.0",
```

Comment:
@jywarren maybe it is because of a version bump. Don't really know how this got added. Any thoughts??

Reply:
@HarshKhandeparkar I would rebase because looking at the commit history I see the image sequencer version was bumped from 2.4.0 to 2.5.0 by jywarren 2 days ago after your commit was made. 

looks-same was actually changed from 6.0.0 to 5.0.1 3 days ago so maybe a rebase will resolve this for you too! 

I think no merge conflicts doesn't mean you don't need a rebase because Git isn't taking into account the updates upstream unless you bring them into your local fork version

Reply:
@sashadev-sky i had rebased the branch but maybe I did something wrong. Also I said there are no merge conflicts since I had rebased it. I just wanted sir to have a look at the changes and maybe suggest what has gone wrong since even I was confused. Don't really know what went wrong. I will try to solve the issue. Thanks!

Reply:
@HarshKhandeparkar try deleting your package.json and rebasing again! Because I saw that there were updates just a couple of hours after your rebase. jywarren is traveling until wednesday so just want to help fix your any version issues before then so you can and him can focus on the code :)